authLdap.php

```diff
@@ -565,9 +563,9 @@ function wp_setcookie($username, $password, $already_md5 = false, $home = '', $s
  * he isn't
  * @conf boolean authLDAP
  */
-function authLdap_show_password_fields($return, $user)
+function authLdap_show_password_fields($user)
 {
-    if (null === $user) {
+    if (is_null($user)) {
```

Comment:
What's the benefit of ```is_null($user)``` compared to ```null === $user```?

As it's not related to the multiple roles I'd rather see this reverted :frowning: